ci/dev/fmt.sh

```diff
@@ -30,6 +30,7 @@ main() {
   doctoc --title '# Maintaining' docs/MAINTAINING.md >/dev/null
   doctoc --title '# Contributor Covenant Code of Conduct' docs/CODE_OF_CONDUCT.md >/dev/null
   doctoc --title '# iPad' docs/ipad.md >/dev/null
+  doctoc --title '# Termux' docs/termux.md >/dev/null
```

Comment:
unrelated to your change, I'm just curious why `--title` is needed with these, I thought it would just read the h1/`#` already in the file?

Reply:
lol honestly...i don't know the answer to that 🤷‍♂️ 

maybe @code-asher has historical context

it does seem like there could be a better way to do this though...

Reply:
No big deal, I imagine that we can tackle that sort of clean up when we make a nice doc site :)